soc/arm/st_stm32/stm32f7/Kconfig.soc

```diff
@@ -13,6 +13,9 @@ config SOC_STM32F723XX
 config SOC_STM32F746XX
 	bool "STM32F746XX"
 
+config SOC_STM32F767XX
```

Comment:
For **F767**, please enable the double precision as it is done for **F769**. Could you keep the way the devices are ordered within this file? F756, F767, F769 etc. 

`select CPU_HAS_FPU_DOUBLE_PRECISION`